rpc/client.go

```diff
@@ -135,20 +135,21 @@ func uploadTests(ctx context.Context, client gopb.CedarPerformanceMetricsClient,
 		}
 		resultData := &gopb.ResultData{
 			Id: &gopb.ResultID{
-				Project:   report.Project,
-				Version:   report.Version,
-				Order:     int32(report.Order),
-				Variant:   report.Variant,
-				TaskName:  report.TaskName,
-				TaskId:    report.TaskID,
-				Mainline:  report.Mainline,
-				Execution: int32(report.Execution),
-				TestName:  test.Info.TestName,
-				Trial:     int32(test.Info.Trial),
-				Tags:      test.Info.Tags,
-				Arguments: test.Info.Arguments,
-				Parent:    test.Info.Parent,
-				CreatedAt: internal.ExportTimestamp(test.CreatedAt),
+				Project:            report.Project,
+				Version:            report.Version,
+				Order:              int32(report.Order),
+				Variant:            report.Variant,
+				TaskName:           report.TaskName,
+				TaskId:             report.TaskID,
+				Mainline:           report.Mainline,
+				Execution:          int32(report.Execution),
+				TestName:           test.Info.TestName,
+				Trial:              int32(test.Info.Trial),
+				Tags:               test.Info.Tags,
+				Arguments:          test.Info.Arguments,
+				Parent:             test.Info.Parent,
+				CreatedAt:          internal.ExportTimestamp(test.CreatedAt),
+				ReferenceVersionID: test.Info.ReferenceVersionID,
```

Comment:
This is the change. Make complains about this line. 
```
➜ make
go build ./ ./rpc ./rpc/internal
# github.com/evergreen-ci/poplar/rpc
rpc/client.go:152:25: cannot use test.Info.ReferenceVersionID (variable of type string) as *string value in struct literal
make: *** [compile] Error 1
```

This is coming from the juniper proto definiton for ReferenceVersionID being optional. It makes this a `*string` instead of a string, and it will not initialize a `*string` from a string literal.

Reply:
I figured this out.